Parsers/ASimWebSession/Parsers/vimWebSessionApacheHTTPServer.yaml

```diff
@@ -0,0 +1,113 @@
+Parser:
```

Comment:
same comments applicable for VIM parser too

Reply:
Check